framework/console/controllers/MessageController.php

```diff
@@ -162,9 +164,10 @@ public function actionExtract($configFile)
      * @param string $sourceMessageTable
      * @param string $messageTable
      * @param boolean $removeUnused
+     * @param boolean $markUnused
      * @param array $languages
      */
-    protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messageTable, $removeUnused, $languages)
+    protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messageTable, $removeUnused, $markUnused, $languages)
```

Comment:
Where should I put the UPGRADE note and how should it look like? I couldn't find anything regrading UPGRADE notes in the contribution docs.